changelogs/head.asciidoc

```diff
@@ -11,6 +11,7 @@ https://github.com/elastic/apm-server/compare/8.5\...main[View commits]
 - `observer.id` and `observer.ephemeral_id` are no longer added to APM documents {pull}9412[9412]
 - `timeseries.instance` has been removed from transaction metrics docs; it was never used {pull}9565[9565]
 - `transaction.failure_count` has been removed. `transaction.success_count` type has changed to `aggregated_metric_double` {pull}9791[9791]
+- `transaction.success_count` has been moved to `event.success_count` {pull}9819[9819]
```

Comment:
The changelog is missing that `event.success_count` has been introduced also for transaction events and transaction metrics. This changelog should contain all changes made to APM Server.

Reply:
Updated changelog, but split the part about transaction events and transaction metrics outside "breaking changes".